src/sema/ast.rs

```diff
@@ -347,7 +347,7 @@ pub struct Function {
     /// For overloaded functions this is the mangled (unique) name.
     pub mangled_name: String,
     /// Solana constructors may have seeds specified using @seed tags
-    pub annotations: Vec<ConstructorAnnotation>,
+    pub new_annotations: ConstructorAnnotations,
```

Comment:
`new_` ?

Reply:
That is a remainder from my development. I will remove it. 🫣